### PR TITLE
feature/86cuwqw1p-cancel-booking-form

### PR DESCRIPTION
**Pull request by ayad0010**
Anusha's code for the cancel booking form. Please review and merge. Thankyou.

### CODE REVIEW

File: behind-the-veil-siteroot/imports/ui/components/pages/cancel-booking/CancelBooking.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        <div className="flex flex-col gap-1">
+          <label for="cancellation-input" className="main-text text-our-black">Reason of cancellation</label>
+          <input id="cancellation-input" className="border-light-grey border-2 p-2 rounded main-text" placeholder="Enter Your Reason"></input>
+        </div>
+
+        {/* button */}
+        <div className="flex gap-10">
+          {/* button */}
+          <div className="flex flex-col gap-4 grow">
+            <Button className="bg-secondary-purple hover:bg-secondary-purple-hover">Cancel Booking</Button>
```
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/ui/components/pages/cancel-booking/CancelBooking.jsx:47]: We might need to have a new ticket for this that later adds the DB functionality for this so we don't forget

File: behind-the-veil-siteroot/imports/ui/components/pages/cancel-booking/CancelBooking.jsx
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/ui/components/pages/cancel-booking/CancelBooking.jsx:1]: is this being called anywhere to view for now? or like a link to the page?

**Review by nethbtju — COMMENTED** [review]
is this page being called anywhere to view for now? or like a link to the page?

**Review by nethbtju — CHANGES_REQUESTED** [review]
Refer to comments for changes needed

File: behind-the-veil-siteroot/imports/ui/components/pages/cancel-booking/CancelBooking.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        <div className="flex flex-col gap-1">
+          <label for="cancellation-input" className="main-text text-our-black">Reason of cancellation</label>
+            <textarea id="cancellation-input" className="border-light-grey border-2 p-2 rounded main-text" placeholder="Enter Your Reason" rows={4} cols={40} />
+        </div>
+
+        {/* button */}
+        <div className="flex gap-10">
+          {/* button */}
+          <div className="flex flex-col gap-4 grow">
+            <Button className="bg-gradient-to-r from-bg-gradient-start to-bg-gradient-end">Cancel Booking</Button>
```
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/ui/components/pages/cancel-booking/CancelBooking.jsx:54]: button colour is purple not gradient

File: behind-the-veil-siteroot/imports/ui/App.jsx [hunk 3]
```diff
@@ -75,8 +77,13 @@ export const App = () => (
...
           <Route path="/service-area" element={<ArtistServiceArea />} />
+
+          {/* TODO:.... */}
+          <Route path="/cancel-booking" element={<CancelBooking/>}/>
+
         </Routes>
       </main>
     </Router>
   </div>
+>>>>>>> develop
```
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/ui/App.jsx:88]: Needs to be removed (my bad sorry)

**Review by nethbtju — CHANGES_REQUESTED** [review]
Minor changes then approved

**Review by nethbtju — APPROVED** [review]
Made minor changes but otherwise vibes

**Review by NikkiShao — APPROVED** [review]
Looks good :)

**Pull request merged**